### PR TITLE
[WIP] Transparent Encryption: CI tests

### PR DESCRIPTION
**Pull request by jrfastab**
testing CI with rolling key updates which should prevent any stalls due to hosts being out of sync and also has cleanup logic to remove old keys which will flush any keys on a node.

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/7451)

### CODE REVIEW

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
:woman_facepalming: forgot to push delay code wasted a test run oops

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
bookInfo failed

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
Runtime FQDN failed

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
Added delay and retry to cilium-health it seems to be failing here but theory we might just not be waiting long enough for ipsec to come online.

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
```
2019-03-21T16:24:43.945874465Z 2019-03-21 16:24:43.945751 I | embed: rejected connection from "127.0.0.1:53512" (error "tls: failed to verify client's certificate: x509: certificate specifies an incompatible key usage", ServerName "")
```

**Comment by jrfastab** [comment]
pulling etcd upgrades.

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
testing CI with retry for cilium-health

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
test-me-please

File: test/helpers/kubectl.go [hunk 1]
```diff
@@ -1940,7 +1940,16 @@ func (kub *Kubectl) ciliumHealthPreFlightCheck() error {
 		return fmt.Errorf("cannot retrieve cilium pods: %s", err)
 	}
 	for _, pod := range ciliumPods {
-		status := kub.CiliumExec(pod, "cilium-health status -o json --probe")
+		var status *CmdRes
+
+		for i := 0; i < 10; i++ {
+			status = kub.CiliumExec(pod, "cilium-health status -o json --probe")
+			time.Sleep(10 * time.Second)
```
Comment (eloycoto) [inline test/helpers/kubectl.go:1947]: I would move this sleep to after the break, and I'll do time.Sleep(i * time.Second)

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
Closing for now we never got this to reliably pass and the rolling update patches are moved. Going to retry with base CI PR.

**Pull request closed without merging**